QuadPaySDK/PaymentWidget/PaymentWidget.swift

```diff
@@ -150,6 +196,13 @@ extension PaymentWidget {
         DispatchQueue.main.async {
             self.timelapseGraphView.drawTimelapseGraph()
         }
+        
+        if(maxFee != 0){
+            paymentWidgetHeaderText.hasFees = "true"
```

Comment:
Use a `boolean` for `hasFees`.  Then this can be simplified to

`paymentWidgetHeaderText.hasFees = maxFee != 0`